.idea/vcs.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
🤔 I have some doubt about removing this file based on [this](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems) however taking a look at [gitignore.io](https://www.toptal.com/developers/gitignore/api/androidstudio,android) they would exclude it.